projects/plugins/jetpack/functions.opengraph.php

```diff
@@ -533,3 +537,118 @@ function jetpack_og_get_description( $description = '', $data = null ) {
 
 	return $description;
 }
+
+/**
+ * Display a Fediverse actor Open Graph tag when the post author has a Mastodon connection.
+ *
+ * @see https://blog.joinmastodon.org/2024/07/highlighting-journalism-on-mastodon/
+ *
+ * @since $$next-version$$
+ *
+ * @param array $tags Current tags.
+ *
+ * @return array
+ */
+function jetpack_add_fediverse_creator_open_graph_tag( $tags ) {
+	/*
+	 * Let's not add any tags when the ActivityPub plugin already adds its own.
```

Comment:
Atomic and Jetpack sites test well per the instructions. 

On Simple sites, do we assume that ActivityPub is active at all times? For some reason, I can't see activityPub specific tags on simple sites. On Jetpack sites, I can see them when the plugin is active. Perhaps, it's not implemented yet for simple sites?

 If we let the ActivityPub plugin handle the tags, should wrap below in a  		` if ! ( defined( 'IS_WPCOM' ) && IS_WPCOM ) `check or do we instead let the Jetpack code take care of adding the tag?

```
add_filter( 'jetpack_open_graph_tags', 'jetpack_add_fediverse_creator_open_graph_tag', 10, 1 );
add_filter( 'jetpack_open_graph_output', 'jetpack_filter_fediverse_cards_output', 10, 1 );
```

Reply:
> On Simple sites, do we assume that ActivityPub is active at all times? For some reason, I can't see activityPub specific tags on simple sites.

The plugin is active indeed, but not available by default until you toggle "Enter the Fediverse" in Tools > Marketing > Connections. Then `wpcom_activitypub_is_active()` starts returning true and the ActivityPub plugin starts outputting its own custom creator tag. At that point, we do not try to add our creator tag in Jetpack.

Could you give this a try and let me know how it goes?

Reply:
Nice, that works. Thanks Jeremy!